packages/jest-expo/package.json

```diff
@@ -5,12 +5,13 @@
   "main": "src",
   "repository": {
     "type": "git",
-    "url": "git://github.com/expo/jest-expo.git",
+    "url": "git://github.com/expo/expo.git",
     "directory": "packages/jest-expo"
   },
   "files": [
     "android",
     "bin",
+    "utils",
```

Comment:
rename to `config`, which is where modules you'd import to set up your Jest config would come from.